.github/workflows/deploy-cloud-functions.yml

```
@@ -0,0 +1,69 @@
+
+env:
+  PROJECT_ID: ${{ secrets.PROJECT_ID }}
+  REGION: us-west1
+  STRAVA_SECRET: ${{ secrets.STRAVA_SECRET }}
+  ENCRYPTION_SECRET: ${{ secrets.ENCRYPTION_SECRET }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)